### ISSUE TITLE
The “Phases” Approach - Preview

### ISSUE THREAD

**Issue by szwnba** [szwnba/affweb#470]
Note - This Approach Is Mainly Recommend For (More) “Evergreen" Campaigns…

And you should know that this approach has actually been tested \- and consistently since 2018, by hundreds of Media Buyers (affiliates, offer owners, advertisers, internal teams, and more), and on thousands of campaigns (& on various Ad Formats & Verticals), averaging anywhere from low 2-Figures & up to high 6-Figures in daily revenue \- and has proven to be much more “worth it” on campaigns that actually have the opportunity & ability to last for years \- a.k.a. not the standard low-value & more-spammy (“Hit & Run”, “Spray & Pray”, “Churn & Burn”) type of campaigns that everyone usually starts their Affiliate Media Buying journeys with.

And you should also know that I always advocate for my fellow Affiliates to focus on “Evergreen” Verticals. And this becomes even more important when you're actually serious about achieving any form of stable & longterm income through Media Buying.

The main Verticals I recommend (right now) are Dating, Casino, & Nutra. And especially if you can add list building (e-mail, sms, push, etc) to your campaign setups.

But, you also can (still) achieve stable & durable campaigns with other verticals, like eCom, InfoProducts, Utility, LeadGen, Bizzop, (& even mVAS), etc, depending on how evergreen the offer’s niche is, and how stable the conversion flow is, and how strong the campaign setup is.

*Note - For making money online in general (whether it's Direct Response, Affiliate Marketing, CopyWriting, SEO, or anything else), I’d mainly recommend focusing on offer niches (& sub-niches) that add actual value in any of the ["3 Core Desires”](https://youtu.be/7M3oOxxHSIc?t=181) (Health, Wealth, & Relationships). And you'd mainly want to focus on conversion flows that don’t depend too much on any specific targeting requirements (like specific carriers, os versions, etc) that can easily affect the performance of the campaign. And the actual (technical) setup of the campaign itself can also increase the stability, durability, & profitability of your campaigns (through email, sms, extra monetization scripts, arbitrage, retargeting, custom api, machine learning, etc), and in some cases even when the niche (and/or flow) wouldn’t normally be considered to be evergreen or stable.

Introduction - The "Phases" Approach - Strategically Using Rules For Testing, Optimizing, & Scaling Campaign Opportunities
--------------------------------------------------------------------------------------------------------------------------

…”Phases" - As in looking at each step of each campaign's “natural” progress & development (a.k.a. the standard “parts” of each campaign’s “life-cycle”), each as their own specific "phase" - or "rounds", "turns", "steps", "periods", "stages", "levels", "part", "chapters", etc - but I recommend you call it whatever makes the most sense to you.

…The main idea here is that each campaign Phase should have it's own specific (& strict) performance goals & rules, and these goals & rules should be (strictly) based on specific amounts of spend (for example, every 10x offer/average payout spend) and/or specific time-frames (for example, every 3 days, or 7 days, or 14 days, or 30 days, etc).

…Approaching your campaigns in phases will help you stay focused on the long term goals & results, while keeping your data clean (which is very important in Media Buying), while also lowering your budget waste (which is very useful, especially for Beginners on tight budget), and especially helps you to avoid over-optimizing & self-sabotaging your campaigns (which is very common, in fact: I consider it the most common mistake Affiliates make).

And, Please Note \- I can promise you that (really) grasping & understanding this approach will have a direct, major, & positive impact on your stability, performance, and overall success as a Media Buyer, both mentally & financially. P.S. Disclaimer: This is not at all a 100% guarantee! Your own success still depends on your own personal circumstance, experience, skills, etc!

…But, Do Note - This approach has been carefully tested, observed, & proven on thousands of campaigns, by hundreds of Affiliates, & even 8-Figure campaigns.

…This is the main (& fundamental) approach that's helped me maintain my own 5-figure/month campaigns since 2018. It's also been proven on pretty much all the types of ad formats & verticals. It's also been used to troubleshoot & revive 6-figure+/month "dead” campaigns.

…It's not only been used by Affiliates, but also Offer Owners, Advertisers, Internal Media Buying Teams (at Trackers, Affiliate Networks, Traffic Sources, & Agencies), & even Teams at Award-Winning Networks.

…And, I’m also constantly being offered deals/jobs/positions from multiple Networks. And I've also had multiple coaching clients that went from being an average Affiliate struggling to make even the most basic campaigns work, to managing 7-Figure+ campaign portfolios for their own clients. And I'd personally give most of the credits to this Phases approach.

..Now, I say all of this only to stress to you how much faith I have in this approach - but you should also know that even after years of helping hundreds of other Affiliates (& these are just the ones I helped directly -I can actually say I've helped even thousands, indirectly), I'm still convinced that only a few people have actually grasped the essence of the approach itself, and even fewer have mastered and/or adapted it properly (& that counts for even some of my best students & clients) - and this is one of my main motivations for putting so much effort into these articles, as an(other) attempt to (re)introduce more Affiliates to what I consider to be essential approaches for mastering (Affiliate) Media Buying.

Ultimately, (as you gain more experience) you'll want to (at least) have your own (personalized) approaches for researching, testing, managing, optimizing, troubleshooting, & scaling your campaigns - a.k.a. your own Media Buying FrameWork.

…For now, I'm just sharing my personal proven framework, which I hope can at least serve as the "foundation" for your own framework.

* Important - Researching Opportunities
---------------------------------------

Research is #1 \- I recommend you take as much time as you need to get confident in the next step to take, even if that means dedicating months to just researching (which is actually something I do, even till this day). The main thing is you should aim to get as much actual data & information as possible from opportunity tools on networks, insight reports, chats with managers, case studies, webinars, journey threads on forums, etc - and as much data & information as you need to give you confidence in what next step to take.

And for me, "that next step" should (actually) feel like FOMO, and for days (& in some cases, even weeks & months). Sometimes I even leave my researched opportunities (a.k.a. notes, data screenshots from managers, traffic sources, networks, etc) open in tabs, that I'll (re)check every now & then. And if I find myself constantly thinking about it (& the thoughts/feeling don't just die out after a few days), then I'll start taking action.

In fact, "putting research first" is how I managed to discover a "safer" way to success (vs what was being promoted in forums, blogs, groups, courses, etc), when I was just starting out with my own journey (which you can [read more about in this thread](https://deadz.rocks/afflft-advrtzr)).

Consider the following advice on Researching as (just) some basic guidelines to help you make some better decisions when looking for your next good campaign opportunity. Also note that this is just my advice on how I personally approach the campaign research process. And I recommend you only use (or adapt) this approach only if it makes sense to you.

And please keep in mind that using an approach like this (still) does not guarantee a successful campaign, and while it does increase the probability for success, in most cases, your campaigns will still result in -ROI at first (& in some cases even stay -ROI for a while).

#### I. The Creative Pairs Found Through Spying

•

Find Creative Pairs That Have Been Running for a Long Time…

*And to be clear - With ”Creative Pairs”, I mean the specific combinations of Images/Videos/Text + Pre-Landers + Offers (& Their Landers/Sales Pages), as in 1 Creative Pair = 1 specific “Image + Text + Pre-lander + Offer Page" Combo…

•

Initially you’d want to find a very specific campaign opportunity - a.k.a. it’s a campaign that’s showing signs of good performance (the basic indication would be multiple long-running creatives & campaigns found on spytools) & that you can easily re-create - to focus on.

•

If you’re just looking for just “any campaign”, I recommend you start your search by using filters for a specific “Traffic Source” or “Affiliate Network” or “Country” (& keep in mind, you should’ve already done some research on Verticals to know which sources, networks & GEOs are considered top performers, otherwise I recommend you [read The “Portfolio” Approach](https://admythic.link/portfolio), first). And if you already know what specific type of campaign(s) you’re going for, you can (also) start off by searching for specific (vertical/niche/angle) creative keywords or specific offer/network/advertiser domains.

•

I recommend you continue to double check multiple filters based on “Traffic Source”, and “Affiliate Network”, and “Country” - And to be clear: I would check each of these filters (one by one) while looking for (more) creatives for the same 1 campaign (found during the initial search). And also check the results by sorting "Longest Running", and "Duration", and "Days Running", and "Date Last Seen" - Again to be clear: I would check each of these sorted results while looking for (more) creatives for the same 1 campaign.

•

Ideally, you'd want to collect folders of creative pairs that have been running for at least 30+ days (once you're more experienced with a Vertical you can also check for 14+ days & 7+ days), and longer should be considered better. And tip - I’d separate these creative folders by Ad Format, Vertical, Niche, Traffic Sources, & Offers.

•

P.S. Even inactive creatives can serve as a good indication for a good campaign opportunity, as long as the campaigns have been running for a longtime (again: 30+ days), and as long as you can find the same (or similar) offer is still active, you can always (re)use these creatives & still get good performance (& note: I've personally re-used 2 year old creatives from spy-tools & still found success).

•

Also important to note - There's plenty of reasons why campaigns die out, but from my experience: In most cases, I find that it has more to do with the person running the campaign (& their budget, skills, circumstances, etc) vs the actual campaign itself being bad (& this can especially be the case with campaigns that have been running for 60-90+ days). And especially when you’re more experienced in a specific Vertical, you can find good performing campaigns, but that others had a hard time getting profitable and/or stable.

•

You can also use [SimilarWeb](https://admythic.link/smwb) on offer domains & network links to find top performing Traffic Types, Traffic Sources, (Countries), & Publishers for specific offers/links. And you might also find more landing pages (also ones that usually don't show up on spytools) by checking top referring links. [Here's an example of this approach](https://admythic.link/smwb-tng).

#### II. The Data/Insights From Affiliate Networks

•

EPC Indications \- Not the most accurate indicator…

•

Keep in mind that the EPC shown on the network is usually way higher than what you'd actually get on your own campaigns, because it's usually based only on clicks to the offer page (& you can’t be sure your funnel will perform the same), and also based on traffic that’s coming from already optimized campaigns (so you usually won’t get the same performance from the start), and also from top performing sources & ad formats (which isn’t always from Media Buying, nor the same sources & formats that you’d run), and from top Affiliates (& you can’t measure their budget, experience, & skill levels vs yours).

•

Optional: Calculate An “Effective EPC” \- Here's a general rule of thumb that I use: Multiply whatever number is shown as the EPC with 0.05 & 0.10 - this is a conservative estimate that assumes you can get at least 5% to 10% of the indicated performance. And (in theory) you'd want this “Effective EPC” amount to be around (& ideally higher) than the average CPC from your Traffic Source.

For example, if your network is indicating an EPC of $1.96 (x 0.05 = $0.098 & x 0.10 = $0.196) so in this case you'd want to aim for a starting bid of around (but ideally lower than) $0.098 - $0.196. And you can check the inventory charts on traffic sources to confirm if the bid is viable. And It's ok if the average bid is a bit higher, since you'll be optimizing your campaigns in the long-run.

In some cases you'll find that the CPC is much higher than your Effective EPC - In these cases I'd recommend beginners to just research a new opportunity, but also note that this doesn't mean it's a bad campaign opportunity, it just means that it might require more skill & experience.

And the Ad Format can also play a big role here. For example, if you're working with Push or Pop campaigns, I'd even suggest you decrease your Effective EPC amount by 50% (because of the low-quality nature of these formats), but if you're working with Natives, Search, Social or Banners, I'd even say you can increase your Effective EPC amount by 50%-100%.

•

Case Studies (& Journey Threads) \- Also usually not the best indicator…

•

Most case studies (& even “journey” threads, depending on the forum) function purely as promotional material for the Networks, & Tools, & Traffic Sources, (& Gurus) involved in them.

•

Every now & then, you might be able to find some good case studies & threads showing a good amount of data (ideally lots of data on hours, bids, zones, browsers, os, isp, etc), and actual creatives, and the offers (or very similar ones) are also still active - if a case study or journey thread has all of these traits, I'd consider it an indication of a good opportunity, and proceed with the researching for (more) creatives on spytools (& again: long-running campaigns would also be an extra confirmation here).

•

In cases where (especially) a case study (& sometimes thread) sounds way to good to be true (& especially if it seems like you can launch that same campaign straight away & hit massive ROI, and especially if there's no clear & good reason or indication why the information is actually being shared so openly (or if & why the campaign has been stopped), then I'd recommend you assume that it's just a faked case study (I actually assume that 95%+ of case studies are just fake).

•

Actual Data & Insights \- This would be the very best indicator that you can get from Affiliate Networks…

•

Some networks (best example is [Monetizer](https://admythic.link/monetizer)) make your job much easier by just sharing their data openly in their opportunities sections (& do keep in mind that this data is usually only applicable to specific Verticals & Ad Formats), but with most networks you'll have a hard time getting actual practical data.

•

The very best scenario would be for you to have a good relationship with your Affiliate Manager, and good enough to the point that they share actual (& specific) active campaign’s data (& creatives) with you.

•

I'd only consider it a good indication IF when you get this data, you actually “have everything you need” to setup the campaign straight away (& you’d usually feel some kind of actual FOMO here), and (optionally) if you're actually able to find (the same/similar) active campaigns on spytools that have been running for at least 7+ days, OR if you actually trust your Affiliate Manager & they actually guarantee you good results (since in some cases, even when you don’t find the campaigns on spytools, they can still actually result in good performance). But beyond all of this, You should also remember that most managers are just bad at their job, and you'd usually want to ignore their recommendations (or at least confirm them for yourself first) & especially if you (or they) are not experienced enough in the specific Vertical.

#### III. The Data/Insights From Traffic Sources

•

Traffic/Inventory Charts & Graphs - Not an accurate indicator.

•

You mainly want to check for an average CPC bid that's below your Effective EPC (explained above). Even if the Traffic Source is the starting point of your research, you should still come back later once you have more data (& calculated your Effective EPC) on potential campaigns/offers.

•

When you know that a specific traffic source is one of the best (like [PropellerAds](https://propellerads.com/), [RevContent](https://admythic.link/revcontent), [MGID](https://www.mgid.com/), [ExoClick](https://www.exoclick.com/), etc) you can also start your research process with the data from Traffic Source itself (vs the Affiliate Network or SpyTools), by making a list of their top countries (& per Ad Format & Device) and researching & matching top-performing campaigns (& their creative pairs) on spytools (following “step I” mentioned above) & networks (following “step II” mentioned above).

•

Tip - For beginners, I'd usually recommend focusing on Tier 3 high-volume & low-bid geo's, at first, since low-volume and/or high-bid geo's can require more budget, experience, time and/or skill to optimize properly.

•

Side-Tip - Also keep in mind that lower quality Ad Formats (like Push & Pops, etc) will just be low-cost in nature, while higher quality formats (like Natives, Banners, Search, Socials, etc) will usually always have (much) higher CPC’s.

…My advice here (even for beginners) would be to focus on higher quality Ad Formats + more Evergreen Offers (for example, [AdCombo](https://admythic.link/adcombo) offers on [MGID](https://admythic.link/mgid)) \- even if it might seem daunting at first, the opportunity costs (& also risks) will be much lower vs focusing on lower-quality campaigns (a.k.a. [”going for 1 Rabbit vs 20 Berries”](https://youtu.be/pkEiHZAtoro)) \- you can find more longterm stability, while also developing better (& more transferable) Media Buying skills with these campaigns (which can also open up more opportunities for you). Read [The "Portfolio" Approach](https://admythic.link/portfolio) for more.

•

Recommendations from Account Managers \- Usually one of the (very) best indicators.

•

Note - Usually you'd only get contacted by an Account Manager (from a Traffic Source) after a certain amount of total spend (or a big deposit) on your account.

…So I’m talking about the more serious & straight to the point “can you get access to this offer” type of Account Managers, not the “your balance is low, please add more funds” type of managers, here…

•

These type of recommendations tend to be much better than the recommendations you'd get from an Affiliate Manager, and especially when they include specific offer recommendations, and the creatives, and a blacklist or whitelist, and bid recommendations. And in some cases, these Account Managers will also do some “magic” on the backend that help your account/campaigns perform better.

•

Tip - My main recommendation here (if you find yourself in contact with one) is to make sure you're getting this information from a senior account manager (or at least one that has 1+ years of experience). And P.S. I always search & check manager's LinkedIn profiles. In general, I wouldn't recommend following campaign recommendations from the standard junior/new account managers (at all, unless you're convinced they're actually good), not even the whitelists, not even campaign settings - from my experience, the only useful information you can get from junior managers is (mainly) global blacklists per GEO & Vertical l(& Ad Format, & Device), but even with these blacklists, it can still be “50-50" scenario if they perform well.

#### \* Reminder - The Overall Best Opportunity Indicators

•

Long-Running Campaigns on SpyTools \- & Again: The longer they're active, means the better the indication of a good opportunity. And if they're still active is a plus, but not a must. And keep in mind, old & in-active creatives can still be good. Again: This is especially the case when it's an Evergreen Vertical.

•

Actual Data & Insights from Affiliate Networks & Managers \- & Again: The main thing here is that they should include actual (targeting, scheduling, & revenue) data. And you should also be able to easily find active campaigns on spytools (that have been running for at least 7+ days) or have good trust built up with your manager.

•

Specific Recommendations from Traffic Source Account Managers \- & Again: The recommendations should be very specific, with specific offers, campaign (targeting and/or scheduling) settings, blacklists, whitelists, & bid recommendations. It should literally feel like the manager is handing you proven campaigns.

And again: These are just some basic guidelines (not strict "must follow" rules) to help you make better decisions when looking for your next campaign opportunity.

And, again: Just to be (extra) clear, this whole process still does not (at all) guarantee that you'll have a successful campaign.

Keep in mind \- Even if you have solid research & a good campaign setup (& even when you get conversions), how successful your campaigns will be will still mainly depend on your media buying skills, experience, budget, and how good you buy & manage your data - a.k.a. how good of an actual Media Buyer you are (or not).

And, this next part will help you with this…

\# 💡- Examples of The Phases Approach
--------------------------------------

### \* Please Note: On These Examples…

•

When I mention ”Your CPA" \- For the following explanation I'll be mentioning "Your CPA" & "Offer Payout" a lot. "Your CPA" is usually just your exact Offer Payout, but in some cases (like with COD or PPS offers or if you have your own sms, e-mail, push, etc setups), you'd want to calculate your own "Ideal CPA" (for example, Offer Payout x Average Approval %, Cost Per (Opt-in) Lead, Cost Per Add to Cart, etc), which is usually lower than the default Offer Payout - so please know that for cases like this, when you see "Your CPA", this means either your "Ideal CPA" or your "Offer Payout", whichever you're using as your main KPI.

•

When I mention "Zone" \- I mean whatever a specific “ad widget location” is called on any traffic source, a.k.a. what PushGround calls "SiteID", EvaDav calls "Source ID", ZeroPark calls "Targets", ExoClick calls "Zone ID" etc.

•

When I mention "Source" \- I mean whatever houses & contains more ad widgets, a.k.a. what PushGround calls "SupplyID, EvaDav calls "Zone ID", ZeroPark calls "Source", ExoClick calls "Site ID", etc.

…And in cases where you're working with a Traffic Source that only uses Zones (and no Sources), you should ignore any specific rules (within a Phase) mentioning "Source".

•

When I mention "Creative Pairs" \- Anytime I mention Creative Pairs, please know that I mean one specific combination of Creative (Media + Text) + Pre-Lander + Offer Page. So, if you have a campaign with 2 creatives + 2 pre-landers + 2 offers, this means you have 8 creative pairs, or if you're direct-linking and you have 2 creatives + 2 offers, this means you have 4 creative pairs. So to be clear - Anytime I mention "Creative Pair", please know that I mean a specific “1 creative (+ 1 lander) + 1 offer” combo.

•

Always Test on Duplicates to Keep Your Data Clean \- I highly recommend you (always) create separate copies of your campaign (and yes, on both the tracker & the traffic source) for each (phase, breakout, and/or split-test) campaign. This will keep your whole campaign portfolio more clean (easier to quickly read & get feedback) & re-usable (easier to use groupings & drill-downs) for the long-run. Also make sure you're using [proper naming conventions](https://funnel.io/blog/how-to-name-your-digital-advertising-campaign-like-a-pro) that are clear to you (& how your brain works & processes data).

•

I Recommend Spending $50 - $100 Minimum on Test Campaigns \- The phases rules & examples in this article are mainly considering offer payouts of $5+. But If you plan to use this approach for low-payout offer campaigns, then I recommend as a general rule that you (always) spend at least $50 - $100 on your initial test campaign, no matter how low the offer payout is.

•

P.S. I don't use or recommend Bayesian "AB Split-Test" Tools \- This is just based on my personal experience. Check [this response from theoptimizer](https://youtu.be/VlzpYnZj8MU?t=3211) for more. But I still recommend you follow your own gut here.

\# Important - These Are The Standard Rules That Apply To Any & All Phases (Always)
-----------------------------------------------------------------------------------

These rules apply can (& should) apply to each Phase to avoid budget waste…

•

Pause Zones With \[1x Your CPA\] Spent & \[0 Conversions\] - You'll want to pause any "budget-wasters" asap. Either by using auto-rules on the traffic source itself or use [theoptimizer](https://admythic.link/theoptimizer). You can always retest these later.

…As long as you're in the earlier Phases, the main goal will be to focus your budget on gather data on top performing Creative Pairs (a.k.a. creative + pre-lander + offer combinations), Targeting (os versions, browsers, carriers, etc) & Scheduling (hours, days of the week, etc), while avoiding as much budget waste as possible.

•

Pause Converting Zones With More Than \[1x Your CPA\] Spent & -50% ROI (or Worse) - You want to give converting -ROI Zones more room to prove themselves before getting paused. This also gives you (or your auto-rules) enough room to experiment with bids on the Zones. If good converting Zones (with 2+ conversions) get paused, you can always re-test them later.

…Again: As long as you're in the earlier Phases, the main goal will be to focus your budget on gather data on top performing Creative Pairs (a.k.a. creative + pre-lander + offer combinations), Targeting (os versions, browsers, carriers, etc) & Scheduling (hours, days of the week, etc), while avoiding as much budget waste as possible.

•

Pause Any Source With \[5x Offer Payout\] Spent & 0 Conversions - You'll want to give Sources more room (vs Zones) to prove themselves, since each Source has multiple (& in some cases a lot of) Zones. But as long as you don't have actual data on best performers (creative pairs and/or targeting and/or scheduling), it's better to stick to strict rules to avoid wasting your budget on under-performers. Usually I'd consider a Source that's spent \[5x Offer Payout\] without any conversions to just be bad & not worth focusing on (I'd expect at the very least 2 conversions after 5x). And keep in mind, (don't worry about missing out on conversions) you can always retest Sources later on & using your best performers.

•

Pause Any Source With \[5x Offer Payout\] Spent & -80% ROI (or Worse) - Ideally (& in general) you'd want to spend at least \[10x Offer Payout\] on a Source before you decide to exclude it forever. But as long as you don't have actual data on best performers (creative pairs and/or targeting and/or scheduling), it's better to stick to strict rules to avoid wasting your budget on under-performers during the earlier phases. Converting Sources (with 2+ conversions) & -ROI can always be re-tested by using your best performers, and in most cases, just using your best performing Creative Pairs (& blacklisting more Zones) will be enough to get underperforming (but still good) Sources profitable. But In some cases you might still have some Sources that will require their own specific Targeting, Scheduling, & Bids (& in some rare cases even their own specific Creative Pairs) in order to get profitable.

…I'd only recommend focusing on underperforming Sources that require more specific optimizations, only if you can confirm that these Sources actually have high-volumes for your campaign (& this would be a good scenario to apply [The “Pools” Approach](https://admythic.link/pools)).

And in Terms of Bids \- And especially for as long as the campaign isn't proven or stable, I recommend you maintain whatever bid is recommended (or average, or minimum required for premium inventory) to get you enough quality traffic (& at least “30% Win-Rate” when this data is available). And it’s very important to not "play" with your bids during testing & optimizing phases, as this can pollute your data (making it harder for you to analyze) & also offset performance.

…If you really want to change bids, I'd even suggest you do this on a new/duplicate campaign (in Tracker + Traffic Source), just to keep your data clean.

\# 🧪 - Phase 1 - The “Testing” Phase
-------------------------------------

…The main goal of Phase 1 is to make sure everything is working properly in your setup, and to validate & prove (to yourself) that your (researched) campaign is actually a good opportunity, while also collecting some initial data to use for Phase 2…

•

Keep a low "Daily Budget" of \[5x Offer Payout\] - \[10x Offer Payout\] - You don't want to overspend on an unproven campaign. The campaign can only be considered a good opportunity if you can prove it to yourself with actual data & results. But also, while your testing any new campaign opportunity, you'd mainly want to avoid "wasting" budget on any bad performing or bot Zones (& Sources).

•

Make Sure Your Campaign Setup Is Working - This is the time to make sure (all) your links (offers, postbacks, tokens, etc) are working correctly. Double-check everything as soon as the first conversion happens.

•

Test Max 6-12 Creative Pairs At A time - After the initial \[5x Offer Payout\] spend on the campaign, you can start pausing the least performing creative pairs.

•

In an “ideal” scenario, you'd want to have \[1x Offer Payout\] spent on each creative pair & at least 10 conversions on the campaign, before you start making decisions to cut pairs. You should prioritize the best performing 1-3 pairs with lowest CPA.

•

Now, if it's an "alright" scenario, and you have \[5x Offer Payout\] Spent & less than 10 conversions, then you should start pausing under-performing pairs based on the average CTR & CPA of your best performing pair, until you have 3 best converting pairs.

…Pairs with better CTR & no conversions (vs your best pair) & still less than \[1x Offer Payout\], can still be included in the campaign.

…Pairs with better CPA & worse CTR (vs your best pair) can still be included, unless this pair has \[1x Offer Payout\] spent & only 1 conversion, and there are other pairs that have more conversions.

•

But if it's a “worst case” scenario, and you have \[5x Offer Payout\] spent on the campaign & 0 conversions, you should start pausing the least performing pairs based on CTR, until you're left with the best 1-3 pairs (based on CTR).

…And once one of these pair gets a conversion, pause all other pairs & just run the one that converted. And if another one converts later on (after its already paused), re-enable it, or only run the one with the lowest CPA out of the two.

…But also note that if after \[5x Offer Payout\] is spent & you have 0 conversions, and you don't feel confident about the campaign, then I'd recommend you research & test a new campaign opportunity, especially if you have a limited budget and/or experience.

•

And keep in mind that in all scenarios, if you notice a specific pair is outperforming all other pairs by 3+ conversions, then I’d recommend you just focus the rest of your (10x Offer Payout) test budget on just that 1 best converting pair.

The main focus of Phase 1 is to “cut down on the variables” asap to focus your budget on collecting the best data possible. It's very important to NOT add any more variables to your campaign during this phase. Focus on cutting, not adding, here.

…The key here is to focus on limiting the Variables in your campaign, and especially the Variables that you have direct control over (a.k.a you can't directly control the quality of traffic), with the goal of increasing your campaign’s (probability for good) performance…

…This will ensure that you don't "pollute" your data \- Adding anything during the Test Phase can "mess up" your stats making it harder for you to make decisions (& also if you're running on a Traffic Source that has some kind of CPA optimization algo, it will throw off the performance of the campaign). If you really want to add new creative pairs, it would be better to wait for the split-test phase (later) or restart the testing phase on a new campaign.

### \# 🔬 - After Phase 1 - \[10x Offer Payout\] Spent

•

Confirm If Your Campaign Is A Good Opportunity…

…And the following is just my personal recommendation (& based on my exp)…

•

Test campaigns that have -30% ROI or better, are easy to optimize \- And usually just require some blacklisting, and focusing on best performers, and/or in some cases bid adjustments, in order to get profitable.

•

Test campaigns that have between -60% & -30% are still ok, but will require more skill \- And in some cases also more budget & time (a.k.a "runway") to optimize & get profitable. And in this case I'd only recommend you continue with this campaign if you really know what you're doing, and only if the campaign is actually worth the effort in terms of volume, profit potential & durability - a.k.a. it's an actual Evergreen campaign & the Traffic Source is actually one of the top performers for the Vertical, etc.

•

But if your campaign has a ROI worse than -60% ROI, then I recommend skipping it \- And proceed with researching & testing new campaigns. It's usually better to spend your budget on finding new campaigns that can perform better from the start vs spending your budget on a campaign that will be hard to optimize (as a deep -ROI campaign like this will probably require more budget & time to get green, vs what it would cost you to find a new & better one). If/Once you have more experience & skill, you can tackle deep red campaigns like these (& only if it's worth the hassle) by drilling-down (& 3 to 5 levels deep) into the data to see if you can find any green segments to proceed with the campaign on.

…And I usually (highly) recommend for beginners (& especially if you're on a tight budget) to focus on researching & finding these campaigns that give -30% ROI+ after the initial (10x Offer Payout) test budget is spent. If you've done all the proper research, and your test campaign still resulted in ROI worse than -30% (& I'd say -40% is the bare-minimum), then I’d suggest skipping that campaign for a new one.

•

Spend Up To \[50x Offer Payout\] Until You Find Enough Zones with 2+ Conversions & Good ROI, and/or Sources With Good Volume & -40%+ ROI, for Whitelist Campaigns…

•

After the initial \[10x Offer Payout\] spent, if you still don't have any specific Zones or Sources with good performance (a.k.a conversions are still scattered across many placements), then you should continue spending (& checking) in \[10x Offer Payout\] batches, until you find enough data on Zones and/or Sources to create Phase 2 “Whitelist" campaigns.

•

You mainly want to make sure that the amount of Zones (and/or Sources) you gather are getting a good amount of traffic, otherwise the next phase in the optimization process will take too much time.

…And in cases where the campaign is just low volume by nature (based on the Traffic Source's available inventory), then you can consider running multiple Phase 2 (and/or breakout) campaigns at the same time & have these campaigns "feed each other" data - a.k.a. running a broad (Targeted and/or Scheduled) campaign that focuses on gathering (more) Zones for the Whitelist campaign, and (at the same time) running a Whitelist campaign that focuses on gathering Targeting & Scheduling data for the other campaign’s settings, etc.

•

Check Your Data for (Targeting and/or Scheduling) Opportunities with 2+ Conversions, Good Volume, & Good ROI for Breakout Campaigns…

•

Check your data for any segments that are outperforming their own counterparts (a.k.a. OS vs OS, Browsers vs Browsers, etc) in terms of volume, conversions, & ROI.

•

For Targeting Data - You'd mainly want to check for any over-performing Os, Os Versions, Browsers, Connection Type, ISP, Cities/States, Languages, & (when available) Specific Traffic Source Targeting (activity, user freshness, inventory type, categories, etc) - mainly anything that the Traffic Source actually allows you to target via their campaign settings.

•

For Scheduling Data - You want to check if you have any specific hours and/or days of the week that are outperforming (or underperforming) vs the others. And you can decide to focus your campaigns only on best performing hours and/or days of the week. Or just exclude the worst performing hours and/or days of the week from your campaigns.

Important \- But you should know that campaign settings don't always work like you'd expect (even when the options are available) on all Traffic Sources.

…In some cases using any Targeting and/or Scheduling settings can conflict with the platforms algorithm, which can cause your campaign(s) to underperform (in weird & "unexplainable" ways).

…I highly recommend always split-testing your (Targeted, & Scheduled, & Targeted + Scheduled) breakout campaigns vs your main (un-targeted & un-scheduled) campaign at first, to confirm if any of the settings aren't messing performance.

…And after you’ve confirmed (for yourself) which settings conflict on a specific Traffic Source, you can confidently decide to apply the Targeting and/or Scheduling settings to all your campaigns or not.

\# 🎯- Phase 2 - The “Focus” Phase
----------------------------------

…The goal of Phase 2 is to use your best performing Creative Pair(s), Targeting, Scheduling, & Zones (& Sources) Data to focus your budget on buying more clean data, and to optimize the campaign(s) to be stable & profitable, while avoiding as much budget waste as possible…

Now, there are multiple possible versions of these Phase 2 campaign(s)…

#### \# 🎯 - Version 1 - Filter Down - Whitelist

•

Whitelist Your Best Performing Zones and/or Sources (So Far) \- Check your data (from Phase 1) for the Zones and/or Sources that have 2+ conversions & good ROI. And check if (you think) have enough Zones and/or Sources with a good amount of volume that you can Whitelist.

…And even after you’ve whitelisted Zones (and/or Sources), you should constantly check for any Zones (and/or Sources) that are underperforming vs the rest in this whitelist, to remove.

•

Keep A Low "Daily Budget" Of \[5x Offer Payout\] - \[10x Offer Payout\] - Especially for Source whitelists, you still don't want to overspend on any bad performing Zones. Once you've actually confirmed good performance you can always increase your budget (or set to unlimited).

•

Run (Only) Your Very Best 1-3 Creative Pair(s) - You still want to keep your "variables" low. And in this case, so you can focus your budget on finding (more) Data on the best performing Target & Schedule settings for your (breakout) campaigns.

#### \# 🎯 - Version 2 - Filter Down - Targeting

•

Target Your Best Performing Targeting Segments \- Check your data (from Phase 1) for any converting OS, OS Versions, Browsers, ISP’s (Carrier/Wifi), Languages, Cities, etc (with 2+ conversions) & that are outperforming their counterparts. And create breakout Targeting campaigns that focus on these specific segments.

…I’d suggest separating campaigns by OS, (& if there’s enough volume: Os Versions, otherwise group best performers together), ISP, & Browsers, (& Traffic Source specific targeting) as this will also give you more control over your bids & allow you to find optimal bids for each campaign based on their own targeting settings.

•

Keep A Low "Daily Budget" Of \[5x Offer Payout\] - \[10x Offer Payout\] - Especially for Source whitelists, you still don't want to overspend on any bad performing Zones. Once you've actually confirmed good performance you can always increase your budget (or set to unlimited).

•

Run (Only) Your Very Best 1-3 Creative Pair(s) - You still want to keep your "variables" low. And in this case, so you can focus your budget on finding (more) Data on the best performing Target & Schedule settings for your (breakout) campaigns.

•

Split-Test \[5x Offer Payout\] of this (Phase 2) campaign vs your main (Phase 1) campaign \- Just in case, to confirm that the new settings don't cause your campaign to underperform. You should be able to tell if the campaign is performing alright within \[5x Offer Payout\] spent

#### \# 🎯 - Version 3 - Filter Down - Scheduling

•

Either: Target The Best Performing Hours (& Days) - If the Phase 1 campaign is still in -ROI. You should check your Phase 1 data to see if you have at least 3+ hours with 2+ conversions & good ROI. And all of these 3+ hours should have around the same ROI (if they're all still -ROI), but ideally these 3+ hours should all be green. And in some cases, you'll find that campaigns perform best on certain days (or even just 1 day) of the week, in these scenarios it might be best to just focus the campaign on just those days (or even just 1 day).

…And you can keep checking your all-time stats (at least every 7 days) to check for any new good performing Hours (gathered via other running campaigns) to add to this version of the Phase 2 campaign.

•

Or: Pause The Worst Performing Hours (& Days) - If the Phase 1 campaign is in +ROI. You can focus on just removing worst performing hours. Ideally you'd want to have a good amount of traffic on each hour (I usually recommend having at least 3x Offer Payout on each hour), before you decide to remove underperforming hours( with 0 conversions and/or bad ROI below -60%). And if you notice any specific day of the week is really underperforming vs the rest, then you can (also) just remove that day from your campaign.

…And you should keep checking your all-time stats (at least every 7 days) to check for any new bad performing Hours to remove from this version of the Phase 2 campaign.

•

Keep A Low "Daily Budget" Of \[5x Offer Payout\] - \[10x Offer Payout\] - You still don't want to overspend on any bad performing Zones & Sources here. Once you've actually confirmed good performance you can always increase your budget (or set to unlimited).

•

Run (Only) Your Very Best 1-3 Creative Pair(s) - You still want to keep your "variables" low. And in this case, so you can focus your budget on finding (more) Data on the best performing Zones, Sources, & Target settings for your (breakout) campaigns.

•

Split-Test \[5x Offer Payout\] of this (Phase 2) campaign vs your main (Phase 1) campaign \- Just in case, to confirm that the new settings don't cause your campaign to underperform. You should be able to tell if the new campaign is performing alright within \[5x Offer Payout\] spent.

*Optional: Additionally, you can also split-test a Phase 2 Campaign that uses both best Targeting + best Scheduling settings. Again: You mainly want to split-test all these different settings vs each to confirm if the settings actually perform & don't cause any performance conflicts.

The main focus of Phase 2 is to optimize (all versions of) your campaigns & make them stable & profitable \- So you can spend as much time & budget as needed on these campaigns until you get your campaigns (more) profitable & stable.

…While giving you the opportunity to test & confirm which campaign settings perform best on the specific Traffic Source \- Which I highly recommend you do, especially if you haven’t already confirmed (for yourself) which campaign settings work best on the specific Traffic Source.

…And if you happen to think that you’ve made any mistake in the optimization of your campaigns in this Phase, you can always “reset" these campaigns by re-analyzing your all-time data (of all your campaign versions grouped together) and drilling & filtering (3 to 5 levels deep) into your data to (re)discover opportunities for new versions of these Phase 2 campaigns.

…Side-Note - Once your Phase 2 campaigns are (more) stable & profitable, you can always (split)test pre-landers & offer pages on any current campaign. And I recommend you do this by sending 10%-20% (max) of your current traffic to “test paths/routes” (via your tracker's campaign settings), & add any good performers (with 2+ conversions & good ROI) to the main path.

\# 🧪 - Phase 3 - The “Re-Testing” Phase (Optional)
---------------------------------------------------

…After you’ve confirmed that your best performing Zones (& Sources), Targeting, & Scheduling data & settings are actually performing well, you can use your best performing campaign settings to re-test (& split-test) to find new Creative Pairs, Zones, & Sources…

#### \# 🧪 - Version 1 - Split-Test - Creatives

•

Whitelist Profitable Zones With 2+ Conversions & 30%+ ROI \- You’ll want to have good number (& as many as possible) of profitable Zones with good volumes (& ideally, gathered from all-time data from all your campaign versions). The more you have = the better.

•

Use Your Best Performing Campaign Settings From Phase 2 \- Duplicate the settings from whichever version of the Phase 2 campaign has proven to be the best performer.

•

Either: Use Your Best Creative’s Image & Test New Texts \- You'll use the best performing Creative as the control (so you should include it in this new campaign as well). And test it’s current Image with new Text combinations that give around same or better CPA & CTR as your current best performing creative.

•

Or: Use Your Best Creative’s Text & Test New Images \- You'll use the best performing Creative as the control (so you should include it in this new campaign as well). And test it’s current Text with new Image combinations that give around same or better CPA & CTR as your current best performing creative.

•

Test Max \[2x Your CPA\] On New Creatives \- Since you're already using your best Zones & Campaign Settings & Best Creative's Data, you can be strict when testing new creatives here. You can start pausing underperforming Creatives (based on CTR) as soon as they have \[1x Your CPA\] spent, and give them up to \[2x Your CPA\] spent (max) to prove themselves. Again: You mainly want to focus on finding new creatives that give around same (& ideally better) CPA & CTR as your current best performing creative.

…And you can proceed to add any new good performing creatives from this campaign to your main campaign(s) from Phase 2.

#### \# 🧪 - Version 2 - Re-Test - Zones

•

Use Your #1 Best Creative Pair \- You'll want to focus on your very best performing Creative Pair (so Creative + Pre-Lander + Offer Page). Optional: You can use the CPA from this Creative Pair as “Your CPA” amount for optimizing Zones in this campaign.

•

Use Your Best Performing Campaign Settings From Phase 2 \- Duplicate the settings from whichever version of the Phase 2 campaign has proven to be the best performer.

•

Re-Test All Zones \- You'll use your best performing Creative Pair & Campaign Settings to retest all Zones (so no lists on this campaign), to focus this campaign on finding any new Zone opportunities.

•

Test Max \[1x Your CPA\] On Zones \- Since you're already using your proven Creative Pairs & Campaign Settings, you can be strict when testing & blacklisting Zones here. You can pause Zones even at specific ROI goal amounts (a.k.a. for 30% ROI goal: \[0.70x Your CPA\], etc), but I'd still recommend testing Zones up to \[1x Your CPA\] just to be extra sure.

…And you can either decide to just run this campaign as your new main blacklist campaign.

…Or you can have this campaign function as a “Zone Finder” campaign, which focuses on finding Zones to add to your current whitelists (or apply [The “Pools” Approach](https://admythic.link/pools) with), while (optional) blacklisting them in this campaign.

…And you can also apply an advanced approach here, by slowly bidding up as you build up the blacklist on this campaign to find even more Zones.

#### \# 🧪 - Version 3 - Re-Test - Sources

•

Use Your #1 Best Creative Pair \- You'll want to focus on your very best performing Creative Pair (so Creative + Pre-Lander + Offer Page). Optional: You can use the CPA from this Creative Pair as “Your CPA” amount for optimizing Sources in this campaign.

•

Use Your Best Performing Campaign Settings From Phase 2 \- Duplicate the settings from whichever version of the Phase 2 campaign has proven to be the best performer.

•

Re-Test All Sources \- You'll use your best performing Creative Pair & Campaign Settings (& optional Zones blacklist) to retest all Sources, to focus this campaign on finding any new Source opportunities (& mainly for separate Whitelist campaigns).

•

Test Max \[5x Your CPA\] On Sources \- Since you're already using your proven Creative Pairs & Campaign Settings, you can be strict when testing & blacklisting Source here. You can pause Sources even around \[3x Your CPA\] spent, but I'd still recommend testing Sources up to \[5x Your CPA\] just to be extra sure.

…And (especially on Traffic Sources that have a lot of big Sources, like DSP’s) you’d mainly want this campaign to function as a “Source Finder” campaign, which focuses on finding new Sources to create separate whitelists campaigns (while also blacklisting your whitelisted Sources in this campaign).

…And (highly recommended) you can also apply an advanced approach here, by slowly bidding up as you build up the blacklist on this campaign to find even more Sources.

#### \# 🧪 - Version 4 - Re-Test - Targeting

•

Whitelist Profitable Zones With 2+ Conversions & 30%+ ROI \- You’ll want to have good number (& as many as possible) of profitable Zones with good volumes (& ideally, gathered from all-time data from all your campaign versions). The more you have = the better.

•

Use Your #1 Best Creative Pair \- You'll want to focus on your very best performing Creative Pair (so Creative + Pre-Lander + Offer Page). Optional: You can use the CPA from this Creative Pair as “Your CPA” amount for re-testing Targeting segments in this campaign.

•

Use Your All-Time Most Profitable Hours (& Days) With 2+ Conversions & 30%+ ROI \- You'll want to use your very best Hours & Days(& ideally, specific best hours for each specific day of the week - if you have enough data) from your all-time stats, & from all your campaigns grouped together.

•

Test Max \[1x Your CPA\] On Targeting Segments \- Since you're already using your best Zones & Campaign Schedule & Best Creative Pairs, you can be strict when re-testing your Targeting settings here. You can pause any segment as soon as they have \[1x Your CPA\].

…And if you confirm any (previously excluded) Targeting settings that's performing well, you can decide to add them to your current/main campaign(s) from Phase 2. Or create a new/duplicate or breakout campaign(s) to apply these new Targeting settings to.

…If the Traffic Source has it’s own specific Targeting settings (like Activities, Inventory Type, User Freshness/Age, etc) then you can also test each of these Targeting options in their own separate campaign (just in case any of their counterparts underperforms vs the other, etc).

#### \# 🧪 - Version 4 - Re-Test - Scheduling

•

Whitelist Profitable Zones With 2+ Conversions & 30%+ ROI \- You’ll want to have good number (& as many as possible) of profitable Zones with good volumes (& ideally, gathered from all-time data from all your campaign versions). The more you have = the better.

•

Use Your #1 Best Creative Pair \- You'll want to focus on your very best performing Creative Pair (so Creative + Pre-Lander + Offer Page). Optional: You can use the CPA from this Creative Pair as “Your CPA” amount for re-testing Targeting segments in this campaign.

•

Use Your All-Time Most Profitable Targeting Settings With 2+ Conversions & 30%+ ROI \- You'll want to use your very best Targeting settings (that you've actually confirmed to be performing well in Phase 2), and ideally by double-checking & confirming with your all-time stats, & from all your campaigns grouped together.

•

Test Max \[1x Your CPA\] On Each Hour (& On Each Day) \- Since you're already using your best Zones & Campaign Settings & Best Creative Pairs, you can be strict when re-testing (mainly) your Hours here. The best scenario would be to have \[1x Your CPA\] spent on each Hour & on each day of the week. And in this case, you'll want to consider data from just this new campaign as your main indicator for new Scheduling opportunities.

…And if you confirm any (previously excluded) Scheduling data that's performing well now, you can decide to either add them to your current/main campaign(s) from Phase 2, or just apply broader (& still good performing) Targeting settings to this campaign, and use this campaign as your new main campaign.

Keep in mind \- These Phase 3 campaigns are completely optional, and you should only consider re-testing (& proving) your data if it makes sense for you & your campaign(s). Testing everything again like this, is mainly (& much more) worth it when/if you're actually focusing on campaigns that can actually last for a longtime - a.k.a. you're actually running Evergreen campaigns.

…If your goal is to just get your campaign profitable & as soon as possible, then there’s probably no need for you to bother with split-testing or re-testing.

…The main goal of Phase 3 is to (re)confirm more opportunities to achieve “max" scale on your current Traffic Source, especially when combined with [The “Pools” Approach](https://admythic.link/pools).

\# 🚀 - Beyond Phase 3 - Scaling (Optional)
-------------------------------------------

Now that you’ve collected (& confirmed) all this data, and you already know what your best performing Creative Pairs (& Targeting & Scheduling Data) are - You can use your best performers to (quickly) test new Traffic Sources (with the same Ad Format) to find new opportunities to (quickly) scale your campaign.

The main things you want to test are on New Sources are…

•

1x Broad (No) Settings Campaign \- And ideally you'd want to (either manually or using [TheOptimizer’s](https://admythic.link/theoptimizer) day-parting rule to) test this campaign on best Day of the Week, & also during best Hours on this specific Day.

•

& 1x “All-Time Best Targeting Settings” Campaign \- Target your best (OS, Os Version, Browser, etc) segments, that the new Traffic Source allows you to target via their campaign settings. There's still no guarantee that these settings will actually work, hence why you’d want to test multiple versions of the campaign at the same time here.

•

& 1x “All-Time Best Scheduling Settings” Campaign \- Target your all-time very best performing Hours, and ideally you’d want to target very best hours on each day. But again: There's still no guarantee that these settings will actually work, hence why you’d want to test multiple versions of the campaign at the same time here.

•

& Optional: 1x “All-Time Best Targeting + Scheduling Settings” Campaign \- You can run a combination of your best Targeting & Scheduling settings campaigns from the start, but you can also test this one later, once you've confirmed that (both the other) campaign settings actually perform well on the Traffic Source. If one of the other campaigns doesn't perform well, I’d say it's probably not worth testing this combination campaign (since the underperforming settings will probably conflict in this campaign), but it's up to you.

•

And Test \[10x Your CPA\] Max On The (New) Traffic Source \- Since you're already using your best performing (& proven) Creative Pairs & Data, you can be aggressive when testing new Traffic Sources. You should be able to confirm good performance (or at least have promising ROI) within \[5x Your CPA\] spent across all the campaign versions. The ideal scenario here would be to test (minimum deposit on) multiple new Traffic Sources at the same time, while also running your main Traffic Source, to (quickly) confirm which new Traffic Source is worth focusing budget on next.

…And you still want to consider your budget when testing new Traffic Sources to maintain your cashflow & to not end up wasting profits from your main Traffic Source.

…And after you’ve confirmed a good opportunity for scaling onto a new Traffic Source, you can proceed by using the same approach (& same rules) you did for Phase 2, without having to start the whole process allover again.

Now, I hope this article helps you in any way.

Keep in mind \- This is just an example of how to strategically approach Testing, Optimizing, & Scaling your campaigns. And this is just what I call “The “Phases” Approach”. Please know that my goal with these approach articles is for them to serve (mainly) as a inspiration point for you to use & build your own Strategic Approach & ultimately your own Media Buying Framework off of (which I highly recommend that you do to ensure success in your journey). But you should still only apply this information if it makes 100% sense to you & your own campaigns, and if it doesn’t please only use the parts that do make sense.

Pro Tip - I recommend you combine this “Phases” Approach with the [The “Pools” Approach](https://admythic.link/pools) for even stronger Media Buying Framework - again, as long as it all makes sense to you.

Want to help motivate AFFMANUAL’s updates? You can always [Buy Me Coffee](https://admythic.link/coffee) ☕️
